package.json

```diff
@@ -6,60 +6,59 @@
   "scripts": {
     "start": "node $NODE_APP_OPTIONS app.js",
     "lint": "node_modules/.bin/eslint .",
-    "format": "node_modules/.bin/prettier-eslint '**/*.js' --list-different",
-    "format:fix": "node_modules/.bin/prettier-eslint '**/*.js' --write",
+    "format": "node_modules/.bin/prettier-eslint $PWD'/**/*.js' --list-different",
+    "format:fix": "node_modules/.bin/prettier-eslint $PWD'/**/*.js' --write",
     "test:unit": "npm run test:unit:_run -- --grep=$MOCHA_GREP",
     "test:acceptance": "npm run test:acceptance:_run -- --grep=$MOCHA_GREP",
     "test:unit:_run": "mocha --recursive --reporter spec $@ test/unit/js",
-    "test:acceptance:_run": "mocha --recursive --reporter spec --timeout 15000 --exit $@ test/acceptance/js",
+    "test:acceptance:_run": "mocha --exit --recursive --reporter spec --timeout 15000 $@ test/acceptance/js",
     "nodemon": "nodemon --config nodemon.json"
   },
   "dependencies": {
-    "async": "^3.1.0",
-    "bcrypt": "^3.0.6",
+    "async": "^3.2.0",
+    "bcrypt": "^4.0.1",
     "body-parser": "^1.19.0",
-    "celebrate": "^10.0.1",
-    "coffeescript": "^1.12.7",
-    "connect-mongo": "^3.0.0",
-    "csurf": "^1.10.0",
+    "celebrate": "^12.0.0",
```

Comment:
I can't find a changelog for this package, but I guess it's OK...

Reply:
`AuthHandlerTests` covers these validation errors, which I considered enough.